.github/workflows/regular.yml

```diff
@@ -1,7 +1,7 @@
 # NOTES:
 # - The map syntax used for matrix is flagged red but actually works
 # - This runs everything in Python 3.10, 3.11 and 3.12
-# - No environments are cached due to space limit
```

Comment:
the old reason is in principle still true so why delete?

Reply:
no it was certainly one of the reasons and still is, if this action creates lots of caches it might delete caches form CI because theres a limit, just keep it and add your reason, no reason to delete